### PR TITLE
Feat/pro 4750/no package manager

### PR DESCRIPTION
**Pull request by patrickvacek**
By the way, this fixes a problem in meta-updater cause by my last OSTree PR (due to using ostreeysroot as a CMake external project fetched with git).

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -40,9 +40,8 @@ set(HEADERS
             logging.h
             openssl_compat.h
             ostree.h
-            ostreefake.h
-            ostreeinterface.h
```
Comment (patriotyk) [inline src/CMakeLists.txt]: Nice. I didn't understand  why  ostreeinterface.h is needed.

File: tests/CMakeLists.txt [hunk 2]
```diff
@@ -37,7 +37,7 @@ add_dependencies(build_tests aktualizr)
 
 
 if(BUILD_SOTA_TOOLS)
-    add_dependencies(build_tests garage-push garage-deploy-deb)
```
Comment (patriotyk) [inline tests/CMakeLists.txt]: This is not mistake?

File: src/CMakeLists.txt (hunk 1 above)
Comment (patrickvacek) [reply src/CMakeLists.txt]: I had grand plans that I realized were pointless, that's all!

File: tests/CMakeLists.txt (hunk 2 above)
Comment (patrickvacek) [reply tests/CMakeLists.txt]: The target doesn't exist anymore. @OYTIS has redone some of the packaging. It's now all done in the root `CMakeLists.txt`.

**Comment by patrickvacek** [comment]
I'm still getting inconsistent crashes in the packagemanager test in CI, so I'm not sure if this is ready to merge yet.

**Comment by patrickvacek** [comment]
Apparently this is an issue with ostreesysroot interacting poorly with code coverage. Still not sure why.

**Comment by patrickvacek** [comment]
Okay, problem solved! Any objections to merging this?

**Comment by cajun-rat** [comment]
None at all

**Pull request merged**